src/validate.js

```diff
@@ -152,8 +156,15 @@ export default function validateFormData(
   formData,
   schema,
   customValidate,
-  transformErrors
+  transformErrors,
+  metaSchema
 ) {
+  //add more schemas to validate against
+  if (!addedMetaSchemas && metaSchema) {
+    ajv.addMetaSchema(metaSchema);
+    addedMetaSchemas = true;
+  }
+
   try {
     ajv.validate(schema, formData);
   } catch (e) {
```

Comment:
Can we add a `console.error` here so if the schema is invalid, the user gets an error?